File: basics/simpleIntrest.java
import java.util.Scanner;

public class simpleIntrest{
public static void main(String args[]){
Scanner s= new Scanner(System.in);

System.out.println("Enter the principle value");
int p=s.nextInt();
System.out.println("Enter the Rate value");
int r=s.nextInt();
System.out.println("Enter the Time value");
int t=s.nextInt();

int si=(p*r*t)/100;
System.out.println("the simple intrest is :"+si);
s.close();
}
}

File: basics/simpleIntrest.cpp
#include<iostream>
using namespace std;

int main(){
    int p,r,t;
    cout<<"Enter the principle value";
    cin>>p;
    cout<<"Enter the rate value";
    cin>>r;
    cout<<"Enter the time value";
    cin>>t;

    int si =  (p*r*t)/100;
    cout<<"SIMPLE INTREST IS:"<<si;

    return 0;
}